src/opendr/planning/end_to_end_planning/__init__.py

```diff
@@ -1,4 +1,6 @@
 from opendr.planning.end_to_end_planning.e2e_planning_learner import EndToEndPlanningRLLearner
-from opendr.planning.end_to_end_planning.envs.UAV_depth_planning_env import UAVDepthPlanningEnv
+import os
+if os.environ.get("ROS_DISTRO") == "noetic":
```

Comment:
we should support melodic too, I don't think there's anything about what you do which would be different so it should work out of the box

Reply:
Yes, I switched it with ROS_VERSION check, instead of ROS_DISTRO.
The reason for this is, the env (UAV_depth_planning_env) relies on rospy, hence, ROS1. It is not necessary for inference, so ROS2 node can run without importing it.